tour_model_eval/first_second_round_evaluation.ipynb

```diff
@@ -0,0 +1,119 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "horizontal-transsexual",
+   "metadata": {},
+   "source": [
+    "## This notebook is to show the evaluation (scatter plot) of two rounds of clustering"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "mighty-ukraine",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import emission.core.get_database as edb\n",
+    "import emission.analysis.modelling.tour_model.get_users as gu\n",
+    "import emission.analysis.modelling.tour_model.load_predict as predict\n",
+    "import matplotlib.pyplot as plt\n",
+    "import emission.analysis.modelling.tour_model.get_plot as plot"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cathedral-pointer",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "participant_uuid_obj = list(edb.get_profile_db().find({\"install_group\": \"participant\"}, {\"user_id\": 1, \"_id\": 0}))\n",
+    "all_users = [u[\"user_id\"] for u in participant_uuid_obj]"
```

Comment:
@corinne-hcr The participant UUID check was specific to the CanBikeCO mini-pilot - it is not true for other programs or datasets. I have created a new mongodump of only the participant data, you should load that instead and change this to the call from `emission.storage.timeseries.abstract_timeseries`